samples/cpp/benchmark_app/README.md

```diff
@@ -114,6 +114,7 @@ Options:
     -i "<path>"               Optional. Path to a folder with images and/or binaries or to specific image or binary file.
                               In case of dynamic shapes models with several inputs provide the same number of files for each input (except cases with single file for any input):"input1:1.jpg input2:1.bin", "input1:1.bin,2.bin input2:3.bin input3:4.bin,5.bin ". Also you can pass specific keys for inputs: "random" - for fillling input with random data, "image_info" - for filling input with image size.
                               You should specify either one files set to be used for all inputs (without providing input names) or separate files sets for every input of model (providing inputs names).
+                              Currently supported data types: bmp, dib, jpeg, jpg, jpe, jp2, png, pbm, pgm, ppm, sr, ras, tiff, tif, bin, npy.
```

Comment:
Readme is not updated with latest changes (If OPENCV is enabled...)

Reply:
Updated!